commit bf643d680056e42e30bb6185b5cb978dd7864e9d (refs/changes/71/5071/3)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-11-20T11:01:54-08:00 (11 months ago)
    
    MORAY-494 node-moray should update fast and artedi dependencies for node-artedi#17
    Reviewed by: Trent Mick <trentm@gmail.com>
    Reviewed by: Kody A Kantor <kody@kkantor.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/CHANGES.md b/CHANGES.md
index 9365f96..b12026f 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## v3.6.0
+
+* [MORAY-494](http://smartos.org/bugview/MORAY-494) node-moray should update fast and artedi dependencies for node-artedi#17
+
 ## v3.5.0
 
 * [TRITON-375](http://smartos.org/bugview/TRITON-375) node-moray should expose similar metrics to moray server
diff --git a/package.json b/package.json
index 276aba8..72f21c1 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "moray",
     "description": "Moray client library",
-    "version": "3.5.0",
+    "version": "3.6.0",
     "author": "Joyent (joyent.com)",
     "keywords": [ "moray" ],
     "main": "./lib/index.js",
@@ -10,14 +10,14 @@
         "url": "git+ssh://git@github.com:joyent/node-moray.git"
     },
     "dependencies": {
-        "artedi": "1.3.0",
+        "artedi": "2.0.0",
         "assert-plus": "^1.0.0",
         "backoff": "^2.4.1",
         "bunyan": "^1.3.2",
         "cmdutil": "^1.1.0",
         "cueball": "^2.3.0",
         "extsprintf": "^1.3.0",
-        "fast": "^2.5.0",
+        "fast": "2.6.0",
         "libuuid": "0.2.1",
         "jsprim": "^1.3.0",
         "posix-getopt": "^1.0.0",
